frontend/src/containers/ContactUs/ContactUs.tsx

```diff
@@ -4,10 +4,10 @@ import { Label, Input, Textarea, Button } from '@components'
 import React, { useState } from 'react'
 import { SubmitHandler, useForm } from 'react-hook-form'
 import { ContactUsForm } from './ContactUs.types'
-import { fetchData } from '@utils'
+import { fetchData, FetchResponse } from '@utils'
```

Comment:
This file has some imports that are not being used anywhere. Please run `npm run lint` before pushing